README.md

```diff
@@ -154,6 +159,15 @@ inside.
     `allNames`. We sort the groups and files, contatenate them, then compare
     that to the list of everything.
 
+- No White Space Specifications
+
+    XCode has a way to, at the user-level, specify customisations for tabs, spaces,
+    indentation amount, and line endings within `Preferences -> Text Editing ->
+    Indentation`.  However, the user might change these preferences at a file or
+    project level, altering the experience for team-members.  As such, we want to
+    keep the file clean of these preferences and rely on `swiftformat` to shape
```

Comment:
Do we want to mention `swiftformat` here? Also, I think we should keep the documentation in the 3rd person.

> ...altering the experience for team-members. Formatting can be enforced using other tools or implementations.

Reply:
Ah, okay.  I am used to the "software paper 'we' second-person" styling from the tech papers at uni.  Can change it.